src/chart/rendering/drawingcontext.h

```diff
@@ -47,8 +50,9 @@ class DrawingContext
 	Painter &painter;
 	const Gen::Options &options;
 	const Styles::Chart &rootStyle;
-	const Events::Draw &rootEvents;
+	const Events &rootEvents;
 	const Layout &layout;
+	std::shared_ptr<RenderedChart> renderedChart;
```

Comment:
this is shared_ptr, but inside Vizzu::Chart is a pure member. Is it must to be a shared_ptr?